modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc

```diff
@@ -5,6 +5,12 @@
 
 This page lists new features added in Redpanda Cloud.
 
+== June 2024
+
+=== Higher connection limits in usage tiers
+
+There is an increased number of supported client connections in all xref:deploy:deployment-option/cloud/byoc-tiers.adoc[tiers]. For example, tier 1 now supports up to 9,000 maximum connections, and tier 9 supports up to 450,000 maximum connections. Connections are regulated per broker for best performance. 
```

Comment:
Wondering if it might be nice for the external search result heading if this read: Redpanda has increased the number of ... (brand would appear in Google results)